README.md

```diff
@@ -32,6 +32,14 @@ pip install .
 
 Add ``-e`` to install in [develop mode](http://pip.readthedocs.org/en/latest/reference/pip_install.html#editable-installs).
 
+``pvlib-python`` comes with Python bindings for NREL's solar position C code,
+however you must manually download the C code directly from [NREL](http://www.nrel.gov/midc/spa/).
+To use the NREL SPA algorithm,
+
+1. Download the SPA files from [NREL](http://www.nrel.gov/midc/spa/)
+2. Copy the SPA files into ``pvlib/spa_c_files``
+3. From this ``pvlib-python`` directory, run ``pip install .``
```

Comment:
For step 3 I would prefer `python setup.py build_ext --inplace` as this is the way described in `pvlib/spa_c_files/README.md` (however I am not sure what happens if you install pvlib afterwards with pip. Is this pip install here needed for pvlib to work with spa, when it is installed?)